src/bus/mod.rs

```diff
@@ -0,0 +1,45 @@
+#[cfg(not(feature = "lattice"))]
+pub(crate) mod inproc;
+#[cfg(feature = "lattice")]
+pub(crate) mod lattice;
+
+#[cfg(not(feature = "lattice"))]
+pub(crate) use inproc::InprocBus as MessageBus;
+
+#[cfg(feature = "lattice")]
+pub(crate) use lattice::DistributedBus as MessageBus;
+
+#[cfg(not(feature = "lattice"))]
+pub(crate) fn new() -> MessageBus {
+    inproc::InprocBus::new()
+}
+
+#[cfg(feature = "lattice")]
+pub(crate) fn new() -> MessageBus {
+    lattice::DistributedBus::new()
+}
+
+pub(crate) fn actor_subject(actor: &str) -> String {
+    format!("wasmbus.actor.{}", actor)
+}
+
+pub(crate) fn provider_subject(capid: &str, binding: &str) -> String {
+    format!("wasmbus.provider.{}.{}", normalize_capid(capid), binding)
```

Comment:
perhaps a comment on why normalize_capid is necessary here?

Reply:
Added.